packages/calcite-components/src/components/shell/shell.tsx

```diff
@@ -129,6 +141,14 @@ export class Shell implements ConditionalSlotComponent {
     );
   }
 
+  renderSheets(): VNode {
+    return (
+      <div hidden={!this.hasSheets}>
```

Comment:
Q: is hidden required here considering slotted elements are rendered only if user parses them.

Reply:
This is because we don't want to display the div if no content is slotted here.